src/equations/numerical_fluxes.jl

```diff
@@ -243,7 +245,7 @@ Base.show(io::IO, numflux::FluxHLL) = print(io, "FluxHLL(", numflux.min_max_spee
 See [`FluxHLL`](@ref).
 """
 const flux_hll = FluxHLL()
-
+const flux_hll_chen_noelle = FluxHLLChenNoelle()
```

Comment:
I did this before because I thought it would look more consistent to the definition of `flux_hll`. But of course I can do it as you suggested here. In that case, I can get rid of `FluxHLLChenNoelle`. However, `min_max_speed_chen_noelle` and `flux_hll_chen_noelle` is required for their strategy. They define special approximate wave speeds

Reply:
That's fine and the way we intended `FluxHLL` to work :+1:

Reply:
Ok great